components/core/src/ffi/ir_stream/encoding_methods.cpp

```diff
@@ -267,6 +267,20 @@ namespace ffi::ir_stream {
 
         bool encode_message (epoch_time_ms_t timestamp_delta, string_view message, string& logtype,
                              vector<int8_t>& ir_buf)
+        {
+            if (!encode_message(message, logtype, ir_buf)) {
```

Comment:
Our internal policy is to spell out the `false ==` to avoid missing the `!` accidentally.